assets/js/front/epoch-front-compiled.js

```diff
@@ -147,7 +147,7 @@ jQuery( document ).ready( function ( $ ) {
              *
              * @since 0.0.1
              */
-            $( app.form_el ).submit( function( event ) {
+            $( document ).on( 'submit', '#' + epoch_vars.form_id, function( event ) {
```

Comment:
@ronalfy At this point it is outdated. It will need a new PR against 1.0.3-dev. Sorry about that.